src/utils/notifyIssues.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+const md = require('markdown-it')()
+
+function getLinkedIssueNumbers({ octokit, prNumber, repoOwner, repoName }) {
+  return octokit
+    .graphql(
+      `
+    query getLinkedIssues($repoOwner: String!, $repoName: String!, $prNumber: Int!) {
+      repository(owner: $repoOwner, name: $repoName) {
+        pullRequest(number: $prNumber) {
+          id
+          closingIssuesReferences(first: 100) {
+            nodes {
+              id
+              number
+            }
+          }
+        }
+      }
+    }
```

Comment:
is there a simpler way to do this with the rest API?

Reply:
I could not find a straight forward alternative in the API docs